features/TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+# [ Feature Title ]
+
+## Problem Summary
+
+> Talk about current goals of the system and how they aren't being met. What outcomes are we not delivering? This can also include an explicit request for improvement that doesn't dictate a specific or concrete solution.
+
+### Assumptions
+
+> This is where you talk about what you assume to be true. This could include assumptions around what users are doing, errors, gaps, etc., based on anecdotes, opinions, testing, or data.
+
+****
+
+## Outcomes
+
+> What are the outcomes you want to achieve? What is the success criteria?
+
+### Metrics
+
+> How are you going to measure the outcome / success?
+
+> **Please provide sample artifact graphs here.**
+
+## Hypothesis
```

Comment:
I suggest that the hypothesis should inform the metrics, and therefore the order of these two items should be switched.